src/platform/test/functional/apps/discover/tabs/_duplication.ts

```diff
@@ -67,7 +67,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await unifiedTabs.duplicateTab(0);
       await discover.waitUntilTabIsLoaded();
       await retry.try(async () => {
-        await expectHistogramHeight(updatedHistogramHeight);
+        await expectHistogramHeight(updatedHistogramHeight2);
```

Comment:
When creating a new tab, the current AppState is carried over.

Reply:
After looking into it, I can confirm this test started failing because of the issue where the chart height was no longer carried over when duplicating a tab. I was able to revert it with the changes in https://github.com/crespocarlos/kibana/pull/9, and keep the intended behaviour.

Reply:
@davismcphee FYI this also happens with the `tabId` in the URL state

Reply:
This issue or the double update one mentioned in https://github.com/elastic/kibana/pull/232237#discussion_r2324724103? In either case, thanks for the heads up. Definitely sounds like a tabs bug, and we'll have to look into it 👍